builtin/client/chatcommands.lua

```diff
@@ -2,18 +2,11 @@
 
 
 core.register_on_sending_chat_message(function(message)
-	if message:sub(1,2) == ".." then
+	if message:sub(1,2) == ".." and message:sub(1,1) == "." then
```

Comment:
the second condition is pointless, if a string starts with `..` it will also start with `.`